src/tracers/blob_saver_tracer.rs

```diff
@@ -53,8 +48,12 @@ fn hash_evm_bytecode(bytecode: &[u8]) -> H256 {
 }
 
 impl Tracer for BlobSaverTracer {
-    fn before_execution(&mut self, _opcode: &Opcode, vm: &mut Execution) -> Result<(), EraVmError> {
-        let current_callstack = vm.current_context()?;
+    fn before_execution(&mut self, _opcode: &Opcode, vm: &mut Execution, _state: &mut VMState) {
+        let current_callstack = vm.current_context();
+        if current_callstack.is_err() {
+            return;
+        }
+        let current_callstack = current_callstack.unwrap();
```

Comment:
```suggestion
        let Ok(current_callstack) = current_callstack. else {
            return;
        };
```

Reply:
Do not commit this suggestion, it won't compile (mind the dot after the current_callstack)

Reply:
Which dot :trollface: